package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unvt/sprite-one",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "description": "Generate sprite images and json without mapnik",
   "main": "dist/lib/index.js",
   "types": "dist/lib/index.d.ts",
```

Comment:
I prefer using `files` property in package.json other than using .npmignore

- about `files` property
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files

also, I think actually only need `dist` folder, `LICENSE`, `README.md` and `package.json` for NPM package, if other files are excluded except those, package size will be significantly reduced.